scripts/defaultFilterList.js

```diff
@@ -0,0 +1,47 @@
+var defaultFilterList = [
+  {
+    src: "http://ajax.googleapis.com/ajax/*",
+    dest: "http://ajax.useso.com/ajax/*",
+    matchProtocol: false,
+  },
+  {
+    src: "http://html5shiv.googlecode.com/svn/trunk/html5.js",
+    dest: "",
+    matchProtocol: true,
+  },
+  {
+    src: "http://fonts.googleapis.com/*",
+    dest: "http://fonts.useso.com/*",
+    matchProtocol: false,
+  },
+  {
+    src: "https://*fastly.net*",
+    dest: "http://*fastly.net*",
+    matchProtocol: true,
+  },
+  {
+    src: "http://*google.*",
```

Comment:
Woa! Some of these filters are too strict! We're not allowed to visit http://translate.google.cn/?hl=en-US?